Sources/ArcGISToolkit/Components/Popups/PopupMedia/Images/AsyncImageView.swift

```diff
@@ -67,7 +73,7 @@ struct AsyncImageView: View {
                     Image(systemName: "exclamationmark.circle")
                         .aspectRatio(contentMode: .fit)
                         .foregroundColor(.red)
-                    Text("An error occurred loading the image: \(error.localizedDescription).")
+                    Text("\(loadingErrorMessage): \(error.localizedDescription).")
```

Comment:
Is there a reason this was broken out, instead of just localizing the whole thing? The placeholder (error.localizedDescription), might be in a different place for some languages.

Reply:
> The placeholder (error.localizedDescription), might be in a different place for some languages.

That's a good point, I've re-included the variable portion.